idd/Energy+.idd.in

```diff
@@ -33137,7 +33137,8 @@ Sizing:System,
   A8, \field System Outdoor Air Method
       \type choice
       \key ZoneSum
-      \key VentilationRateProcedure
+      \key Standard62.1VentilationRateProcedure
+      \key Standard62.1SimplifiedProcedure
```

Comment:
I don't believe there is any precedent for using a "." (period) inside a key choice name.  Strikes me as odd.

Reply:
Yeah, I agree, it is a bit odd. Output:Table:SummaryReports forged the way a while back with `Standard62.1Summary`. But it is the only one in the entire IDD.
@Myoldmopar ?